pkg/osquery/table/mdm.go

```diff
@@ -145,7 +161,20 @@ func getMDMProfileStatus() (profileStatus, error) {
 	}, nil
 }
 
-type profileStatus struct {
-	DEPEnrolled  bool
-	UserApproved bool
+func getDEPStatus() (depStatus, error) {
```

Comment:
what this function appears to be checking is if the device is on a OS version where the OS supports these flags. I don't think there's actual correlation between this command having 3 lines of output and DEP profile status. 

I would solicit more output from different systems and update this helper to more reliably print a status.